fluent-package/yum/systemd-test/update-to-next-version-service-status.sh

```diff
@@ -65,9 +65,9 @@ else
 fi
 
 if [ "$status_before_update" = active ]; then
-    # The service should restart automatically after update
+    # The service should NOT restart automatically after update
```

Comment:
I'd like to more comments like this to clarify.

```suggestion
    # The service should NOT restart automatically after update
    # (The process before update should continue to run)
```